readme.md

```diff
@@ -1056,6 +1056,26 @@ module.exports = {
 
 Note: Next.js will automatically use that prefix in the scripts it loads, but this has no effect whatsoever on `/static`. If you want to serve those assets over the CDN, you'll have to introduce the prefix yourself. One way of introducing a prefix that works inside your components and varies by environment is documented [in this example](https://github.com/zeit/next.js/tree/master/examples/with-universal-configuration).
 
+### Transpile NPM modules
```

Comment:
As this is just exposing the `webpack` API would the documentation make more sense to say that, and use this as an example of how you might do it?